lib/news_scraper/trainer/uri_trainer.rb

```diff
@@ -2,12 +2,16 @@ module NewsScraper
   module Trainer
     class UriTrainer
       def initialize(uri)
-        @article_scrape_patterns = Transformers::Article.scrape_patterns
+        @article_scrape_patterns = YAML.load_file(NewsScraper::Constants::SCRAPE_PATTERN_CONFIG_FILE)
         @data_type = NewsScraper::Trainer::PresetsSelector.new(uri)
 
         @uri = uri
         @raw_html = Extractors::Article.new(uri: uri).extract
-        @transformer = Transformers::Article.new(uri: uri, payload: @raw_html)
+        @transformer = Transformers::Article.new(
```

Comment:
So, I didn't remove scrape patterns from the initializer yet, I think the transformers article will need a closer look at to remove it from there as scrape_details also relies on it.. its a bit of a smelly initializer tbh and the class deserves a bit of a refactor IMO